src/utils.ts

```diff
@@ -44,4 +44,57 @@ export class NullOutstreamStringWritable extends stream.Writable {
             callback();
         }
     }
-};
\ No newline at end of file
+};
+
+export const checkIfEnvironmentVariableIsOmitted = (key: string): boolean => {
+
+    const omitEnvironmentVariables: string [] = [
+        'LANG',
+        'HOSTNAME',
+        'PWD',
+        'HOME',
+        'PYTHON_VERSION',
+        'PYTHON_PIP_VERSION',
+        'SHLVL',
+        'PATH',
+        'GPG_KEY',
+        'CONDA',
+        'AGENT_TOOLSDIRECTORY',
+        'GITHUB_WORKSPACE',
+        'RUNNER_PERFLOG',
+        'RUNNER_WORKSPACE',
+        'RUNNER_TEMP',
+        'RUNNER_TRACKING_ID',
+        'RUNNER_TOOL_CACHE',
+        'DOTNET_SKIP_FIRST_TIME_EXPERIENCE',
+        'JOURNAL_STREAM',
+        'DEPLOYMENT_BASEPATH',
+        'VCPKG_INSTALLATION_ROOT',
+        'PERFLOG_LOCATION_SETTING'
+    ];
+
+    const omitEnvironmentVariablesWithPrefix: string [] = [
+        'JAVA_',
+        'LEIN_',
+        'M2_',
+        'BOOST_',
+        'GOROOT',
+        'ANDROID_',
+        'GRADLE_',
+        'ANT_',
+        'CHROME',
+        'SELENIUM_',
+        'INPUT_'
+    ];
+    for (let i = 0; i < omitEnvironmentVariables.length; i++){
+        if (omitEnvironmentVariables[i] === key.toUpperCase()){
+            return true;
+        }
+    }
+
+    let matched = omitEnvironmentVariablesWithPrefix.filter((prefix: string) => key.toUpperCase().startsWith(prefix) );

```

Comment:
you should be able to use `some` function here

Reply:
Sure